src/Reducer.js

```diff
@@ -48,6 +48,11 @@ function inject(state, action, props, scenes) {
             case POP_ACTION:
                 assert(!state.tabs, "pop() operation cannot be run on tab bar (tabs=true)")
                 return {...state, index:state.index-1, children:state.children.slice(0, -1) };
+            case POP_TO_TOP:
+                assert(state.tabs, "Parent="+state.key+" is not tab bar, popToTop action is not valid");
+                let child = state.children[state.index];
+                let newChild = {...child, ...{children: [child.children[0]], index:0}};
+                return {...state, ...{children:[...state.children.slice(0,state.index), newChild, ...state.children.slice(state.index+1)]}};
```

Comment:
In my case I need it for the tab bar. The behaviour is based off the apple app store. Tapping the tab on which you already are is back to root of the tab. Is there a cleaner way to implement this? I could put it in the reducer of course but I thought it was a nice addition to the actions.